### ISSUE TITLE
Scoped package names fail

### ISSUE THREAD

**Issue by Vestride** [vadimdemedes/trevor#56]
When using a [scoped package](https://docs.npmjs.com/misc/scope), trevor fails.

`.travis.yml`
```
language: node_js
node_js:
  - '7'
```

`package.json`
```json
{
  "name": "@foo/bar",
}
```
Results in this error message.

```
✖ 7: error
Error: Command failed: docker rmi test-@foo/bar-7
Error response from daemon: Error parsing reference: "test-@foo/bar-7" is not a valid repository/tag: invalid reference format

    at Promise.all.then.arr (/usr/local/lib/node_modules/trevor/node_modules/execa/index.js:201:11)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

If I change the node version to `'6'`, the error says `"test-@foo/bar-6"`.

If I change the package name to `"name": "foo-bar"`, trevor starts building the environment as expected.

**Comment by vadimdemedes** [comment]
Good catch, thanks for reporting!

**Comment by vadimdemedes** [comment]
Fixed in `2.3.0`.
